### PR TITLE
[PORT][9.0] product_links/product_multi_link

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

File: product_multi_link/README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+To use this module, you need to:
+
+* Go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/113/9.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (pedrobaeza) [inline product_multi_link/README.rst]: Don't put this text here.

File: product_multi_link/README.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+These types of links are common in e-commerce shops.
+
+It can be used as a base to implement synchronizations with
+e-commerce (for instance, it is used in magentoerpconnect).
+
+Usage
+=====
+
+To use this module, you need to:
```
Comment (pedrobaeza) [inline product_multi_link/README.rst]: This is intended as a template for the usage instructions. If you don't put instructions (although it's better to put them), please remove this paragrah

File: product_multi_link/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2011 Guewen Baconnier,Camptocamp,Elico-Corp,Sodexis
```
Comment (pedrobaeza) [inline product_multi_link/__openerp__.py]: Put a line for each contributor and its year. For example, you are making the contribution on 2016, so you have to put:

```
# © 2016 Sodexis
```

**Comment by atchuthan** [comment]
@pedrobaeza changes done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by oscarolar** [comment]
I tried to test on runbot but it seems to be a problem, Logged as admin I can't see the menu installed by the app, also noticed that the sales App is not installed, Am I missing something?
missingmenu

**Comment by pedrobaeza** [comment]
@oscarolar, you have to give permissions to admin, because by default they're not granted.

**Comment by atchuthan** [comment]
@oscarolar, This module adds a field in Product Variant form.
So, we need to enable Technical Settings / Manage Product Variants and Sales access in the runbot.

**Comment by guewen** [comment]
:+1:

**Comment by atchuthan** [comment]
@pedrobaeza I have added a PR for module name change at OCA/OpenUpgrade#462

**Pull request merged**